packages/react-reconciler/src/ReactFiberTreeReflection.js

```diff
@@ -117,11 +117,19 @@ export function findCurrentFiberUsingSlowPath(fiber: Fiber): Fiber | null {
   let b = alternate;
   while (true) {
     let parentA = a.return;
```

Comment:
What if parentA here is a Suspense boundary with no fragment and its alternate is a Suspense boundary with a fragment?

In that case the children won't be representing the same thing. However, they also will never be equal so maybe it doesn't matter?

Reply:
That was my thinking, at least